css-values/Overview.bs

```diff
@@ -1088,27 +1185,93 @@ Absolute lengths: the ''cm'', ''mm'', ''q'', ''in'', ''pt'', ''pc'', ''px'' unit
 		    <th>name
 		    <th>equivalence
 	<tbody dfn-type=value dfn-for="<length>">
+		<tr><th><dfn id="m">m</dfn>
+		    <td>meters
+		    <td>1m = 1000mm
+		<tr><th><dfn id="dm">dm</dfn>
+		    <td>decimeters
+		    <td>1dm = 100mm
 		<tr><th><dfn id="cm">cm</dfn>
 		    <td>centimeters
-		    <td>1cm = 96px/2.54
+		    <td>1cm = 10mm
 		<tr><th><dfn id="mm">mm</dfn>
 		    <td>millimeters
-		    <td>1mm = 1/10th of 1cm
+		    <td>1mm = 96px/25.4 
+		<tr><th><dfn id="mu">mu</dfn>
+			<!--'um'?-->
+		    <td>micrometers (&mu;m), "microns"
+		    <td>1mu = 1mm/1000
 		<tr><th><dfn id="q">Q</dfn>
-		    <td>quarter-millimeters
-		    <td>1q = 1/40th of 1cm
+		    <td>quarter-millimeters (&#x30ad;&#x30a5;)
+		    <td>1q = 1mm/4
+		<tr><th><dfn id="cc">cc</dfn>
+		    <td>ciceros
+		    <td>1cc = 12dd = 18q
+		<tr><th><dfn id="dd">dd</dfn>
+		    <td>didots
+		    <td>1dd = 3mm/8 = 1.5q
+		<tr><th><dfn id="thm">thm</dfn>
+			<!--tm, th, thb, tb, nn, ii, inm "metric inch"-->
+		    <td>thumbs
+		    <td>1thm = 25mm
+		<tr><th><dfn id="mdl">mdl</dfn>
+			<!--nn, mod, ftm "metric foot"-->
+		    <td>modules
+		    <td>1mdl = 300mm = 12thm
```

Comment:
New unit `dot` or `emu` for the greatest common denominator of the smallest units supported by CSS(3). Mentioned in #315.